components/navigation.css

```diff
@@ -0,0 +1,84 @@
+.header-bar {
+  align-items: center;
+  justify-content: space-around;
+  background-color: var(--white-color);
+  color: var(--cta-color);
+  width: 100%;
+  padding: 1rem;
+}
+
+.header-left {
+  width: 10rem;
+  justify-content: space-around;
+  align-items: center;
+}
+
+.web-logo {
+  align-items: center;
+  color: var(--btn-primary-color);
+  gap: 0.5rem;
+  font-size: 2rem;
+}
+
+.web-logo-icon {
+  font-size: 2.5rem;
+}
+
+.header-mid {
+  background-color: var(--white-color);
+  color: var(--cta-color);
+  width: 20rem;
+  border: 0.1rem solid var(--cta-color);
+  border-radius: 0.4rem;
+  padding: 0.2rem;
+}
+
+.search-bar {
+  width: 100%;
+  border: none;
+}
+
+.header-right {
+  width: 22rem;
+  align-items: center;
+  justify-content: space-between;
+  gap: 2rem;
+  width: 30%;
+}
+
+.cart {
+  font-size: 1rem;
+  cursor: pointer;
+  gap: 0.5rem;
+  align-items: center;
+}
+
+.search-bar:focus {
+  outline: none;
```

Comment:
Instead of outline:none you can give outline:2px solid transparent; so that can avoid accessibility issues